distributions/JavaScript/parse_headers.py

```diff
@@ -0,0 +1,68 @@
+import os
+
+include_directory = os.fsencode(".././include")
+function_types = ["int", "double", "long", "void"]
+function_names = []
+inc_file_names = []
+function_counter = 0
+
+
+def browse_files(directory):
+    for file in os.listdir(directory):
+        filename = os.fsdecode(file)
+        inc_file_names.append(filename)
+        function_names.append(parse_file(".././include/" + filename))
+
+
+def parse_file(filename):
+    result = []
+
+    with open(filename, "r") as file:
+        lines = file.readlines()
+
+    for line in lines:
+        split_line = line.split()
+        if len(split_line) >= 2 and split_line[0] in function_types:
+            if split_line[1][0] == "A":
+                result.append(split_line[1][:-1])
+
+    return result
+
+
+def create_script(script_name):
+    i = 0
+
+    with open(script_name, "w") as file:
+        write_intro(file)
```

Comment:
@marwin1991 @adudzik Where can I find implementation of `write_intro` ?
```
→ python -m flake8 parse_headers.py                                                                          ~/CLionProjects/library/distributions/JavaScript
parse_headers.py:36:9: F821 undefined name 'write_intro'
parse_headers.py:49:9: F821 undefined name 'write_outro'
```

Reply:
I think it is some lagacy code becasue in the past this script had generated whole compile_to_js.sh file then we changed it to only generate funs

Reply:
Can you fix it ?

Reply:
Done